go/vt/wrangler/reparent.go

```diff
@@ -457,9 +457,6 @@ func (wr *Wrangler) plannedReparentShardLocked(ctx context.Context, ev *events.R
 		reparentJournalPos = rp
 	}
 
-	remoteCtx, remoteCancel = context.WithTimeout(ctx, waitSlaveTimeout)
```

Comment:
Was this unused? I don't see `remoteCtx` used below either, but just verifying the reason for removal.

Reply:
Yes, it was unused. Not sure when references to it were removed.

Reply:
It turns out that when code was moved around in #5226, it led to us calling PromoteSlave with an already used context, effectively reducing the timeout to some indeterminate value, and also partly undoing the changes in #4850. However, this is all still on the feature branch, so no harm done. I have added the necessary lines of code to get it back to the way it is supposed to be.